### DISCUSSION TITLE
discussions completion for `author:` hangs Chrome for github.com

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#41708] [Discussions] [answered]
### Select Topic Area

Bug

### Body

<img width="707" alt="image" src="https://user-images.githubusercontent.com/2119212/207556621-5547254f-7145-4c17-bdc7-b50339cfac98.png">

This is a regression, it used to work reasonably well.

I imagine it's trying to process all contributors to:
https://github.com/community/community/discussions

But that's not a small number and that's something the engine should know and should avoid doing. I've tripped on this a couple of times.

Note that the most likely case is that I'm searching for:
1. myself
2. someone i've searched for in the past

**Comment by LindseyB** [comment, marked as answer]
Thanks for the feedback, we recently released a fix for this yesterday. Does this resolve your problem or is it still happening?

**Reply by jsoref** [reply]
Thanks. Fixed.
